src/Runner.Worker/action_yaml.json

```diff
@@ -32,10 +32,10 @@
         "outputs": {
             "mapping": {
                 "loose-key-type": "non-empty-string",
-                "loose-value-type": "outputs-attributes"
+                "loose-value-type": "output-definition"
```

Comment:
Avoiding new terms here like "attributes" and instead using "definition" which aligns with the parser classes (technically schema classes)